trunner/testcase.py

```diff
@@ -89,11 +91,11 @@ def sysexec(self, proc):
     def exec_test(self, proc):
         try:
             # Wait for a prompt
-            # On armv7a9-zynq7000-qemu jffs initialization may take to ~20s, that's why it's set to 25
+            # On armv7a9-zynq7000-qemu jffs initialization may take to ~25s, that's why it's set to 30
```

Comment:
We are moving in wrong direction :face_with_head_bandage:

Reply:
The ash start-up takes some time, that's why it's increased here. This problem is known and it's planned to resolve it, so I've created issue (https://github.com/phoenix-rtos/phoenix-rtos-project/issues/450) for that and mentioned it in the comment.